modules/virt-configuring-secondary-network-vm-live-migration.adoc

```diff
@@ -0,0 +1,86 @@
+
+To configure a dedicated secondary network for live migration, you must first create a bridge network attachment definition for a namespace in the CLI. Then add the name of the `NetworkAttachmentDefinition` object to the `HyperConverged` custom resource (CR).
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* The Multus Container Network Interface (CNI) plug-in is installed on the cluster.
+* Every node on the cluster has at least two Network Interface Cards (NICs), and the NICs to be used for live migration are connected to the same VLAN.
+* The virtual machine (VM) is running with eviction strategy `LiveMigrate`.
+
+.Procedure
+
+. Create a `NetworkAttachmentDefinition` manifest.
++
+.Example configuration file
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: my-secondary-network <1>
+  namespace: openshift-cnv
+spec:
+  config: '{
+    "cniVersion": "0.3.1",
+    "name": "migration-bridge",
+    "type": "macvlan",
+    "master": "eth1", <2>
+    "mode": "bridge",
+    "ipam": {
+      "type": "whereabouts", <3>
+      "range": "10.200.5.0/24" <4>
+    }
+  }'
+----
+<1> The name of the `NetworkAttachmentDefinition` object.
+<2> The name of the NIC to be used for live migration.
+<3> The name of the CNI plug-in that provides the network for this network attachment definition.
+<4> The IP address range for the secondary network. This range must not have any overlap with the IP addresses of the main network.
+
+. Open the `HyperConverged` CR in your default editor by running the following command:
++
+[source,terminal]
+----
+oc edit hyperconverged kubevirt-hyperconverged -n openshift-cnv
+----
+
+. Add the name of the `NetworkAttachmentDefinition` object to the `spec.liveMigrationConfig` stanza of the `HyperConverged` CR. For example:
++
+.Example configuration file
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  liveMigrationConfig:
+    completionTimeoutPerGiB: 800
+    network: my-secondary-network  <1>
+    parallelMigrationsPerCluster: 5
+    parallelOutboundMigrationsPerNode: 2
+    progressTimeout: 150
+...
+----
+<1> The name of the Multus `NetworkAttachmentDefinition` object to be used for live migrations.
+
+
+. Save your changes and exit the editor. The `virt-handler` pods restart and connect to the secondary network.
+
+.Verification
+
+* When the node that the virtual machine runs on is placed into maintenance mode, the VM automatically migrates to another node in the cluster. You can verify that the migration occurred over the secondary network and not the default pod network by checking the target IP address in the virtual machine instance (VMI) metadata.
++
+[source,terminal]
+----
+oc get vmi <vmi-migratable> -o jsonpath='{.status.migrationState.targetNodeAddress}'
```

Comment:
What is `<vmi-migratable>` referring to? Does the user need to substitute it with their own parameter?

Reply:
It is the name of the VMI, to be replaced by the user with their own value. Maybe `<vmi_name>` is better suited here?